<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClassForm</class>
 <widget class="QWidget" name="CreateClassForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание класса</string>
  </property>
  <widget class="QLineEdit" name="cr_cl_class_name_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cr_cl_create_class_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать класс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cr_cl_info_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
  </widget>
  <widget class="QLabel" name="class_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cr_cl_protocol_name_lineedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="protocol_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Протокол</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
